README.md

```diff
@@ -312,10 +312,26 @@ reference constants. You'll need to explicitly load `app/bot`, then:
 ```ruby
 # config/initializers/bot.rb
 unless Rails.env.production?
-  Dir["#{Rails.root}/app/bot/**/*.rb"].each { |file| require file }
+  bot_files = Dir[Rails.root.join('app', 'bot', '**', '*.rb')]
```

Comment:
@elct9620 One small change, just rename `bot` to `bots` so, it matches with the `application.rb` folder.